src/main/flight/mixer.c

```diff
@@ -609,9 +609,15 @@ void FAST_CODE mixTable()
     }
 }
 
-int16_t getThrottlePercent(void)
+int16_t getThrottlePercent(bool useScaled)
 {
-    int16_t thr = (constrain(rcCommand[THROTTLE], PWM_RANGE_MIN, PWM_RANGE_MAX ) - getThrottleIdleValue()) * 100 / (motorConfig()->maxthrottle - getThrottleIdleValue());
+    int16_t thr = constrain(rcCommand[THROTTLE], PWM_RANGE_MIN, PWM_RANGE_MAX); 
+    
+    if (useScaled) {
+        thr = (thr - getThrottleIdleValue()) * 100 / (motorConfig()->maxthrottle - getThrottleIdleValue());
+    } else {
+        thr = (thr - 100) / 10;
```

Comment:
1000 surely ?

Reply:
thr = (thr - PWM_RANGE_MIN) * 100 / (PWM_RANGE_MAX - PWM_RANGE_MIN);

Reply:
@breadoven Yes, should be -1000, typo.

@tiriad no need to scale. It is contstrained to 1000 - 2000, which the RC input should already be set to.

Reply:
Yes, I only suggested it to follow the style of other variables and the code prior to the modification assuming it was correct. https://github.com/iNavFlight/inav/blob/5510259ec3fb255dbf1a14b6660adb2aaf50679e/src/main/io/osd.c#L879